packages/rocketchat_theme/client/imports/components/header.css

```diff
@@ -34,9 +34,7 @@
 	&__block {
 		display: flex;
 
-		margin: 0 -0.5rem;
-
-		padding: 0 0.5rem;
+		margin: 0 1rem 0 0;
```

Comment:
All the header blocks look horizontally misaligned.
<img width="518" alt="captura de tela 2019-02-18 as 11 11 19" src="https://user-images.githubusercontent.com/2263066/52956482-0bdd9180-336e-11e9-8c3e-de605235e2bd.png">

Reply:
@tassoevan Okay, I will change that too